### ISSUE TITLE
BGP sessions are not established for both IPv4 and IPv6 due to IPv4&6 addresses on all Port channel lost after config reload

### ISSUE THREAD

**Issue by stephenxs** [Azure/sonic-buildimage#3043]
**Description**
After issuing a "config reload" ipv4&6 addresses on all PortChannels have been lost in kernel protocol stack. "ifconfig PortChannelxxx" shows only ipv6 link local address existing.
It seems that thing goes well on SONiC side:
intfmgr has handled the CONFIG_DB update and issued the "ip address add xxx dev PortChannelxxx" command (which will add address to kernel). according to the log, no error is reported for this command and no command that delete address has been issued.
orchagent and sai also work well. the newly added ip address has been popagated to APPL_DB and SAI side.
I guess there might be some race conditions.
probability of reproducing: less than 20%.

**Steps to reproduce the issue:**
in testbed with topo t0 or t1-lag, execute:
`sudo config reload`
the config_db.json is a default config file for t0 or t1-lag topo.

**Describe the results you received:**
ip address on all PortChannels have been lost, only ipv6 link local address existing.
admin@mtbc-sonic-01-2410:/proc/net$ sudo ifconfig PortChannel0001
PortChannel0001: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9100
        inet6 fe80::9a03:9bff:fef3:f500  prefixlen 64  scopeid 0x20<link>
        ether 98:03:9b:f3:f5:00  txqueuelen 1000  (Ethernet)

**Describe the results you expected:**
after config reload, the ip addresses should not be lost.

**Additional information you deem important (e.g. issue happens only occasionally):**
***reproducing log
```
admin@mtbc-sonic-01-2410:~$ sudo ifconfig PortChannel0001
PortChannel0001: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9100
        inet 10.0.0.56  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fc00::71  prefixlen 126  scopeid 0x0<global>
        inet6 fe80::9a03:9bff:fef3:f500  prefixlen 64  scopeid 0x20<link>
        ether 98:03:9b:f3:f5:00  txqueuelen 1000  (Ethernet)
        RX packets 14524  bytes 1612055 (1.5 MiB)
        RX errors 0  dropped 333  overruns 0  frame 0
        TX packets 13954  bytes 2182785 (2.0 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

admin@mtbc-sonic-01-2410:~$ sudo config reload 
Clear current config and reload config from the file /etc/sonic/config_db.json? [y/N]: y
Running command: systemctl stop dhcp_relay
Running command: systemctl stop swss
Running command: systemctl stop snmp
Warning: Stopping snmp.service, but it can still be activated by:
  snmp.timer
Running command: systemctl stop lldp
Running command: systemctl stop pmon
Running command: systemctl stop bgp
Running command: systemctl stop teamd
Running command: systemctl stop hostcfgd
Running command: /usr/local/bin/sonic-cfggen -j /etc/sonic/config_db.json --write-to-db
Running command: systemctl restart hostname-config
Running command: systemctl restart interfaces-config
Running command: systemctl restart ntp-config
Running command: systemctl restart rsyslog-config
Running command: systemctl restart swss
Running command: systemctl restart bgp
Running command: systemctl restart teamd
Running command: systemctl restart pmon
Running command: systemctl restart lldp
Running command: systemctl restart snmp
Running command: systemctl restart dhcp_relay
Running command: systemctl restart hostcfgd
admin@mtbc-sonic-01-2410:~$ sudo ifconfig PortChannel0001
PortChannel0001: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9100
        inet6 fe80::9a03:9bff:fef3:f500  prefixlen 64  scopeid 0x20<link>
        ether 98:03:9b:f3:f5:00  txqueuelen 1000  (Ethernet)
        RX packets 41  bytes 3138 (3.0 KiB)
        RX errors 0  dropped 7  overruns 0  frame 0
        TX packets 4  bytes 520 (520.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

syslog
```
Jun 19 10:02:51.035309 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:51.035309 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doTask: exit
Jun 19 10:02:51.035309 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> select: enter
Jun 19 10:02:51.681907 mtbc-sonic-01-2410 INFO teamd#supervisord: teamsyncd Listens to teamd events...
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:42,042 INFO spawned: 'vrfmgrd' with pid 112
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:43,049 INFO success: vrfmgrd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:45,904 INFO spawned: 'vlanmgrd' with pid 126
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:46,940 INFO success: vlanmgrd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:47,010 INFO exited: swssconfig (exit status 0; expected)
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:49,736 INFO spawned: 'intfmgrd' with pid 140
Jun 19 10:02:51.745225 mtbc-sonic-01-2410 INFO swss#supervisord 2019-06-19 10:02:50,820 INFO success: intfmgrd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< select: exit
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doTask: enter
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfGeneralTask: enter
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfGeneralTask: exit
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.021283 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 10:02:52.040019 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.56/31 dev PortChannel0001 :
Jun 19 10:02:52.040019 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.040019 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.040019 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip -6 address add fc00::71/126 dev PortChannel0001 :
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfGeneralTask: enter
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0002 is ready
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfGeneralTask: exit
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.073195 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0002 is ready
Jun 19 10:02:52.083545 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.58/31 dev PortChannel0002 :
Jun 19 10:02:52.083545 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.083545 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.083545 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0002 is ready
Jun 19 10:02:52.119386 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip -6 address add fc00::75/126 dev PortChannel0002 :
Jun 19 10:02:52.120393 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.120526 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfGeneralTask: enter
Jun 19 10:02:52.123136 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0003 is ready
Jun 19 10:02:52.123825 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfGeneralTask: exit
Jun 19 10:02:52.124076 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.125452 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0003 is ready
Jun 19 10:02:52.163362 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.60/31 dev PortChannel0003 :
Jun 19 10:02:52.163362 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.163362 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.163362 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0003 is ready
Jun 19 10:02:52.182286 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip -6 address add fc00::79/126 dev PortChannel0003 :
Jun 19 10:02:52.182286 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.182286 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfGeneralTask: enter
Jun 19 10:02:52.182286 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0004 is ready
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfGeneralTask: exit
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0004 is ready
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.62/31 dev PortChannel0004 :
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 10:02:52.213611 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0004 is ready
```

related dmesg
```
admin@mtbc-sonic-01-2410:~$ grep "PortChannel0001" dmesg
[46297.024714] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0001: link becomes ready
[56339.574728] PortChannel0001: Port device Ethernet112 removed
[56397.246093] PortChannel0001: Mode changed to "loadbalance"
[56397.318158] IPv6: ADDRCONF(NETDEV_UP): PortChannel0001: link is not ready
[56399.230209] 8021q: adding VLAN 0 to HW filter on device PortChannel0001
[56434.337725] PortChannel0001: Mode changed to "loadbalance"
[56434.372192] IPv6: ADDRCONF(NETDEV_UP): PortChannel0001: link is not ready
[56434.372195] 8021q: adding VLAN 0 to HW filter on device PortChannel0001
[56447.968123] PortChannel0001: Port device Ethernet112 added
[56460.822282] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0001: link becomes ready
admin@mtbc-sonic-01-2410:~$ grep "PortChannel0002" dmesg
[46296.304255] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0002: link becomes ready
[56339.583838] PortChannel0002: Port device Ethernet116 removed
[56397.633153] PortChannel0002: Mode changed to "loadbalance"
[56397.653491] IPv6: ADDRCONF(NETDEV_UP): PortChannel0002: link is not ready
[56399.230232] 8021q: adding VLAN 0 to HW filter on device PortChannel0002
[56434.463961] PortChannel0002: Mode changed to "loadbalance"
[56434.490241] IPv6: ADDRCONF(NETDEV_UP): PortChannel0002: link is not ready
[56434.490245] 8021q: adding VLAN 0 to HW filter on device PortChannel0002
[56448.076919] PortChannel0002: Port device Ethernet116 added
[56460.122970] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0002: link becomes ready
admin@mtbc-sonic-01-2410:~$ grep "PortChannel0003" dmesg
[46296.839472] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0003: link becomes ready
[56339.592798] PortChannel0003: Port device Ethernet120 removed
[56397.997851] PortChannel0003: Mode changed to "loadbalance"
[56398.035810] IPv6: ADDRCONF(NETDEV_UP): PortChannel0003: link is not ready
[56399.230253] 8021q: adding VLAN 0 to HW filter on device PortChannel0003
[56434.574230] PortChannel0003: Mode changed to "loadbalance"
[56434.591797] IPv6: ADDRCONF(NETDEV_UP): PortChannel0003: link is not ready
[56434.591801] 8021q: adding VLAN 0 to HW filter on device PortChannel0003
[56448.242233] PortChannel0003: Port device Ethernet120 added
[56459.551531] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0003: link becomes ready
admin@mtbc-sonic-01-2410:~$ grep "PortChannel0004" dmesg
[46298.202531] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0004: link becomes ready
[56339.602515] PortChannel0004: Port device Ethernet124 removed
[56398.293188] PortChannel0004: Mode changed to "loadbalance"
[56398.487626] IPv6: ADDRCONF(NETDEV_UP): PortChannel0004: link is not ready
[56399.230274] 8021q: adding VLAN 0 to HW filter on device PortChannel0004
[56434.676795] PortChannel0004: Mode changed to "loadbalance"
[56434.701905] IPv6: ADDRCONF(NETDEV_UP): PortChannel0004: link is not ready
[56434.701910] 8021q: adding VLAN 0 to HW filter on device PortChannel0004
[56448.346006] PortChannel0004: Port device Ethernet124 added
[56459.100765] IPv6: ADDRCONF(NETDEV_CHANGE): PortChannel0004: link becomes ready
```
    **Output of `show version`:**

    ```
admin@mtbc-sonic-01-2410:~$ show version 

SONiC Software Version: SONiC.HEAD.7-d67c6d4b
Distribution: Debian 9.9
Kernel: 4.9.0-8-2-amd64
Build commit: d67c6d4b
Build date: Sun Jun 16 09:18:35 UTC 2019
Built by: johnar@jenkins-worker-4

    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by stephenxs** [comment]
I found sth strange when the ip address on portchannels lost. See blow.

A config reload procedure when ip addresses are lost, teamd has been stopped/started twice, which, I suspect, cause the ip addresses just set lost.
Like the following:
Stop teamd
Start teamd
Set ip address on portchannel
Stop teamd again
Start teamd again, I suspect teamd will destroy and recreate all the portchannels in the kernel, causing all addresses lost.

However, in a normal config reload procedure, teamd has been stopped/started only once, when ip addresses will not lost.

The whole log has been attached to the issue in the github.

The following logs are extracted from the entire log.
Jun 19 13:44:41.488929 mtbc-sonic-01-2410 INFO config: 'reload' executing...
Jun 19 13:44:41.505509 mtbc-sonic-01-2410 INFO systemd[1]: Stopping DHCP relay container...
……
Jun 19 13:44:42.385943 mtbc-sonic-01-2410 INFO teamd.sh[9100]: 0
Jun 19 13:44:42.386664 mtbc-sonic-01-2410 INFO teamd.sh[16395]: teamd
Jun 19 13:44:42.396459 mtbc-sonic-01-2410 INFO systemd[1]: Stopped TEAMD container.
……
Jun 19 13:45:46.933919 mtbc-sonic-01-2410 INFO teamd.sh[19030]: Starting existing teamd container with HWSKU ACS-MSN2410
……
Jun 19 13:45:53.536802 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:44:42,101 INFO stopped: teammgrd (terminated by SIGTERM)
Jun 19 13:45:53.536802 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:44:42,102 INFO stopped: teamsyncd (terminated by SIGTERM)
……
Jun 19 13:45:56.773855 mtbc-sonic-01-2410 NOTICE teamd#teammgrd: :- main: --- Starting teammrgd ---
Jun 19 13:46:00.785188 mtbc-sonic-01-2410 INFO teamd#supervisord: teamsyncd Listens to teamd events...
Jun 19 13:46:01.802225 mtbc-sonic-01-2410 INFO teamd#supervisord: start.sh teamsyncd: started
……
Jun 19 13:46:01.807595 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 13:46:01.807595 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfGeneralTask: exit
Jun 19 13:46:01.807595 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 13:46:01.807595 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 13:46:01.815651 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.56/31 dev PortChannel0001 :
Jun 19 13:46:01.816901 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 13:46:01.816901 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
Jun 19 13:46:01.823711 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 13:46:01.840233 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip -6 address add fc00::71/126 dev PortChannel0001 :
Jun 19 13:46:01.842216 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
……
//here, teamd has been stopped and started again, which may cause address lost.
Jun 19 13:46:31.519784 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:46:21,879 WARN received SIGTERM indicating exit request
Jun 19 13:46:31.519902 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:46:21,879 INFO waiting for teamsyncd, rsyslogd, teammgrd to die
Jun 19 13:46:31.519989 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:46:21,880 INFO stopped: teammgrd (terminated by SIGTERM)
Jun 19 13:46:31.520066 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:46:21,881 INFO stopped: teamsyncd (terminated by SIGTERM)
……
Jun 19 13:46:33.221164 mtbc-sonic-01-2410 NOTICE teamd#teammgrd: :- main: --- Starting teammrgd ---

**Comment by stephenxs** [comment]
what the config reload procedure regarding team stuff looks like in a normal procedure:
teammgrd only stop/start once

team stuff stopped
```
Jun 19 13:35:05.373187 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:33:57,053 INFO waiting for teamsyncd, rsyslogd, teammgrd to die
Jun 19 13:35:05.373187 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:33:57,054 INFO stopped: teammgrd (terminated by SIGTERM)
Jun 19 13:35:05.373187 mtbc-sonic-01-2410 INFO teamd#supervisord 2019-06-19 13:33:57,055 INFO stopped: teamsyncd (terminated by SIGTERM)

```
teammgrd started
```
Jun 19 13:35:08.739228 mtbc-sonic-01-2410 NOTICE teamd#teammgrd: :- main: --- Starting teammrgd ---
```
intfmgrd started
```
Jun 19 13:35:13.929179 mtbc-sonic-01-2410 NOTICE swss#intfmgrd: :- main: --- Starting intfmgrd ---
```
setting ip addresses
```
Jun 19 13:35:13.953187 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- isIntfStateOk: Lag PortChannel0001 is ready
Jun 19 13:35:13.973237 mtbc-sonic-01-2410 INFO teamd#supervisord: start.sh teamsyncd: started
Jun 19 13:35:13.985168 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :- exec: /sbin/ip address add 10.0.0.56/31 dev PortChannel0001 :
Jun 19 13:35:13.985168 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :< doIntfAddrTask: exit
Jun 19 13:35:13.985168 mtbc-sonic-01-2410 DEBUG swss#intfmgrd: :> doIntfAddrTask: enter
```
no teammgrd restarting for the second round found.

**Comment by stepanblyschak** [comment]
Fix in https://github.com/Azure/sonic-buildimage/pull/3114

**Comment by stephenxs** [comment]
> Fix in #3114

By deploying code in the [#3114](https://github.com/Azure/sonic-buildimage/pull/3114), intfMgrd creating ip address ahead of teamd started for the second time is effectively prevented.
But I have a further question, why teamd docker stopped/started twice during a single config reload and whether it can be avoided.

**Comment by stephenxs** [comment]
the issue that some docker stopped/started twice has been fixed by [[config] Do not stop or restart dependent services when reloading config #582](https://github.com/Azure/sonic-utilities/pull/582).
